content/backend/graphql-js/5-connecting-server-and-database.md

```diff
@@ -272,6 +270,23 @@ The Prisma CLI also uses information that's provided in `.graphqlconfig.yml`. Th
 
 <Instruction>
 
+Update `prisma.yml` to include a deploy hook:
+
+```bash(path=".../hackernews-node/database/prisma.yml")
+endpoint: `https://eu1.prisma.sh/public-graytracker-771/hackernews-node/dev
+datamodel: datamodel.graphql
+
+# Deploy hook
+hooks:
+  post-deploy:
+    - graphql get-schema --project database
+```
+</Instruction>
+
+A deploy hook is invoked when Prisma is done with deploying. In this case, we want to download the schema using the `get-schema` command and pointing to the `database` project that was configured in `.graphqlconfig.yml`.
```

Comment:
I really like this explanation! The only things I'd slightly adjust is to make clear that the `graphql get-schema` command is based on the [GraphQL CLI](https://oss.prisma.io/content/graphql-cli/01-Overview.html) and probably add an instruction to install the CLI globally.